nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/resources/templates/index.html.hbs

```diff
@@ -675,6 +676,14 @@
             </div>
             <div id="system-diagnostic-endpoints" class="section-endpoints hidden"></div>
         </div>
+        <div class="section">
+            <div class="section-header">
+                <div class="title link">Versions</div>
+                <div class="sub-title section-description">Versioning</div>
```

Comment:
Let's try a different one that's a little descriptive here, like "Control flow versioning" or "Save, delete manage flow versioning" or something better?

Reply:
Thanks for the review, @zenfenan! Updated the pull-request addressing your comments.